les/vflux/server/prioritypool.go

```diff
@@ -206,14 +198,14 @@ func (pp *PriorityPool) RequestCapacity(node *enode.Node, targetCap uint64, bias
 	}
 	c, _ := pp.ns.GetField(node, pp.ppNodeInfoField).(*ppNodeInfo)
 	if c == nil {
-		log.Error("RequestCapacity called for unknown node", "id", node.ID())
+		log.Error("requestCapacity called for unknown node", "id", node.ID())
 		return math.MaxInt64, false
 	}
 	var priority int64
 	if targetCap > c.capacity {
-		priority = c.nodePriority.EstimatePriority(targetCap, 0, 0, bias, false)
+		priority = c.nodePriority.estimatePriority(targetCap, 0, 0, bias, false)
 	} else {
-		priority = c.nodePriority.Priority(targetCap)
+		priority = c.nodePriority.priority(targetCap)
```

Comment:
If the requested capacity is lower than the current one, should we just return "YES"?

Reply:
Actually there is no need to return any kind of priority value any more because the capacity logic was simplified. `setCap` parameter is also not used anymore because there is the capacity curve to efficiently serve pool queries. So I simplified it, now it only returns the final capacity.